website/static/js/fangorn.js

```diff
@@ -1270,6 +1271,30 @@ function _fangornTitleColumn(item, col) {
 }
 
 /**
+ * Defines the contents of the modified column (does not include the toggle and folder sections
+ * @param {Object} item A Treebeard _item object for the row involved. Node information is inside item.data
+ * @param {Object} col Options for this particular column
+ * @this Treebeard.controller
+ * @returns {Array} Returns an array of mithril template objects using m()
+ * @private
+ */
+function _fangornModifiedColumn(item, col) {
+    var tb = this;
+    if (item.data.isAddonRoot && item.connected === false) { // as opposed to undefined, avoids unnecessary setting of this value
+        return _connectCheckTemplate.call(this, item);
+    }
+    var assert = require('assert');
+    if (item.kind === 'file' && item.data.permissions.view && item.data.modified) {
+        return m(
+            'span',
+            // "new Date" required for non-ISO date formats
+            new Moment(new Date(item.data.modified)).format('YYYY-MM-DD hh:mm A')
```

Comment:
That's how I did it originally but I got a jshint error from Travis without the "new"